### PR TITLE
Adding an e2e test for verifying https-only annotation with kubemci

### PR DESCRIPTION
**Pull request by nikhiljindal**
Fixes https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/142

Refactoring existing GCE ingress test to be reusable with kubemci.
Similar to https://github.com/kubernetes/kubernetes/pull/61871

cc @G-Harmon @nicksardo @madhusudancs 

```release-note
NONE
```

### CODE REVIEW

**Comment by G-Harmon** [comment]
It says 7 failed and 1 pending checks. please fix..
for example: test/e2e/network/ingress.go:123:36: undefined: jib

**Comment by nikhiljindal** [comment]
All tests are passing now

File: test/e2e/network/ingress.go [hunk 1]
```diff
@@ -741,6 +735,20 @@ func executePresharedCertTest(f *framework.Framework, jig *framework.IngressTest
...
 
+func executeStaticIPHttpsOnlyTest(f *framework.Framework, jig *framework.IngressTestJig, ipName, ip string) {
+	jig.CreateIngress(filepath.Join(framework.IngressManifestPath, "static-ip"), f.Namespace.Name, map[string]string{
+		framework.IngressStaticIPKey:  ipName,
+		framework.IngressAllowHTTPKey: "false",
+	}, map[string]string{})
+
+	By("waiting for Ingress to come up with ip: " + ip)
+	httpClient := framework.BuildInsecureClient(framework.IngressReqTimeout)
+	framework.ExpectNoError(framework.PollURL(fmt.Sprintf("https://%v/", ip), "", framework.LoadBalancerPollTimeout, jig.PollInterval, httpClient, false))
```
Comment (madhusudancs) [inline test/e2e/network/ingress.go]: Is it possible to make the format string `%s` that way it's more obvious what you are trying to do?

**Review by madhusudancs — COMMENTED** [review]
/lgtm
/approve

File: test/e2e/network/ingress.go (hunk 1 above)
Comment (nikhiljindal) [reply test/e2e/network/ingress.go]: Done

**Comment by nikhiljindal** [comment]
Thanks for the review @madhusudancs 
Updated code as per comments

**Comment by madhusudancs** [comment]
Thanks!

/lgtm

**Pull request merged**